runtime/cudaq/qis/qvector.h

```diff
@@ -52,10 +52,14 @@ class qvector {
     getExecutionManager()->initializeState(targets, vector.data());
   }
 
+  qvector(const std::vector<double> &vector) {
```

Comment:
should this be `qvector(const std::vector<double>& vector) : qvector(std::vector<complex>{vector.begin(), vector.end()}) {}` ?

Reply:
Good catch. Thanks.